### ISSUE TITLE
Content Addition: Accessibility on VA.gov Page - Research Pilot Sessions with CAIA A11YS

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#88273]
## Description

>[!NOTE]
> **The following needs added on the page for Accessibility on VA.gov:** 
> - **Page Section:** Accessibility Resources by Discipline / Resources for Researchers /[**Get Help**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-on-va-gov#AccessibilityonVA.gov-Gethelp
): 
>     - **New Content:** [Conducting a pilot session with a CAIA accessibility specialist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) (recommended), before running your research sessions, to mitigate any potential accessibility-related issues.
>     - **Placement of Content:** See below "Revised Version" for location recommendation for the additional content. 

#### Get help

**The CAIA team can support your inclusive research sessions by:**

- Reviewing your research plan
- Providing technical support during sessions with assistive technology (AT) users
- Observing sessions with AT users from an accessibility point of view and providing findings
- Put in an [AT Research Support Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&projects=&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) with CAIA to get started.

#### Get help

**The CAIA team can support your inclusive research sessions by:**

- Reviewing your research plan
- [Conducting a pilot session with a CAIA accessibility specialist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) (recommended), before running your research sessions, to mitigate any potential accessibility-related issues. 
- Providing technical support during sessions with assistive technology (AT) users
- Observing sessions with AT users from an accessibility point of view and providing findings
- Put in an [AT Research Support Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&projects=&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) with CAIA to get started.

## Relevant URL
- **Page Content Location:** Accessibility Resources by Discipline / Resources for Researchers /**Get Help**
- **Page URL:** https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-on-va-gov#AccessibilityonVA.gov-Gethelp

## Which type of team are you on? (Platform team, VFS team, or Leadership)

- **VFS Team:** Sitewide CAIA - Accessibility Team
- **Slack Channel:** [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH)
- **OCTO A11Y Lead:** Martha Wilks @artsymartha68 
**cc:** @coforma-jamie @NaomiPMC @naomimarcussen

**Comment by jknipes** [comment]
Hi @sara-amanda, thanks for your content suggestions. The Platform Content team will review your request and put it in our backlog to refine for next steps.

**Comment by jknipes** [comment]
@sara-amanda this issue has been resolved and we are closing this ticket as _Done_
